client/overview/ProductControls.jsx

```diff
@@ -1,20 +1,50 @@
 import React from 'react';
 import './overview.css';
+import axios from 'axios';
 
+import StyleSelector from './StyleSelector.jsx';
 import utils from './overviewUtils.js';
 import RatingStars from '../reviews/RatingStars.jsx';
-import StyleSelector from './StyleSelector.jsx';
+import ratingHelper from '../reviews/reviewHelpers.js';
+
 
 class ProductControls extends React.Component {
   constructor(props) {
     super(props);
 
-    if (props.style) {
-      console.log('this.props.style.original_price:', utils.toCurrency(this.props.style.original_price));
+    this.renderPrice = this.renderPrice.bind(this);
+  }
+
+  // async getStarAverage() {
```

Comment:
I see this function is commented out. Since I am already making this api call in my component, it is another piece of information that we could have at the top level.